README.md

```diff
@@ -228,12 +228,21 @@ curl -F "data=@<Path to Evaluation File>" -F "filename=<Filename To Show in Heim
 
 ### How to Install
 
-If you would like to change Heimdall to your needs, Heimdall has 'Development Mode' you can use, where if you make changes to the code, the app will automattically rebuild itself and use those changes. Please note that you should *not* run development mode when deploying Heimdall for general usage. To get started on a Debian-based distribution, follow these steps:
+If you would like to change Heimdall to your needs, Heimdall has 'Development Mode' you can use, where if you make changes to the code, the app will automatically rebuild itself and use those changes. Please note that you should *not* run development mode when deploying Heimdall for general usage. To get started on a Debian-based distribution, follow these steps:
```

Comment:
Awkward grammar; works but consider changing for syntactical clarity.

Reply:
This wording works. Added (except for the Debian reference, since I could not get the given instructions working on Debian).